1-mandatory/2-quotegenerator/index.html

```diff
@@ -8,10 +8,30 @@
       integrity="sha384-Vkoo8x4CGsO3+Hhxv8T/Q5PaXtkKtu6ug5TOeNV6gBiFeWPGFN9MuhOf23Q9Ifjh"
       crossorigin="anonymous"
     />
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.2.1/css/all.min.css" integrity="sha512-MV7K8+y+gLIBoVD59lQIYicR65iaqukzvf/nwasF0nqhPay5w/9lJmVM2hMDcnK1OnMGCdVK+iQrJ7lzPJQd1w==" crossorigin="anonymous" referrerpolicy="no-referrer" />
     <link rel="stylesheet" href="style.css" />
   </head>
   <body>
     <!-- Write your HTML in here -->
+    <main>
+      <div class="container">
+        <div class="quotes">
+           <h1><i class="fa-solid fa-quote-left"></i></h1>
+        <h1 id="quote"></h1>
```

Comment:
You can include `fa-quote-right` in here, like this:

```
<h1>
   <i class="fa-solid fa-quote-left"></i>
   <span id="quote"></span>
   <i class="fa-solid fa-quote-right"></i>
</h1>
```

After that, in you javascript, simply update innerText of the quote:

```
function getQuote () {
   let newPick = pickFromArray(quotes)
  
   quoteTitle.innerText = `${newPick["quote"]}`
   authorTitle.innerText = `-${newPick["author"]}`;
} 
```

Reply:
thank you